### DISCUSSION TITLE
Create a language learning solution

### DISCUSSION THREAD

**Discussion by venkatsriraman** [Sunbird-Ed/Community#297] [Sunbird ED configuration discussion]
Need -
1. Authoring of content such as video, audio, epub, pdf, and question sets to be packaged as courses.
2. Consumption of these courses on some app
3. Usage data capturing and visualization
4. Capability to take audio input from users as part of consumption
5. Include Vakyansh (speech recognition) in the consumption

Will be great, to have this flow configured for an English module available here : https://clixplatform.tiss.edu/eb-unit0-english-beginner/course/content/ . Happy to sit along and get a few Chapters setup and configured in Ed.

**Comment by Krishnaj20** [comment]
thanks @venkatsriraman for initiating this thread. Can you let us know what is the use case for points 4 and 5?

@kameshbhr @luckynvdu
